app/models/thredded/user_topic_follow.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module Thredded
+  class UserTopicFollow < ActiveRecord::Base
+    REASON_MANUAL = 'manual'
```

Comment:
@glebm agree maybe should have done it that way in the first place, but it works right now and other than saving a bit of space in the db doesn't seem to have a compelling need?